docs/UserGuide.md

```diff
@@ -32,80 +32,173 @@
+  The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `list`<br>
+  `setevent 2 complete milestone 1 sd/1-10-2016 st/1400 ed/1-10-2016 et/1600`<br>
+  Set an event on 1st October 2016 starting from 1400 - 1600 to the 2nd task in the task manager.
+* `find Highlight`<br> 
+  `setevent 1 Highlight roadshow sd/23-10-2016 ed/23-10-2016`<br>
+  Set a full day event on 23rd October 2016 to the 1st task in the results of the `find` command.
+  
+#### Set task as completed : `setcomplete`
+Sets an event to the specified task from the task manager.<br>
+Format: `setcomplete TASK_INDEX`
+
+> Sets the task at the specified `TASK_INDEX` as complete.<br>
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+  
+Examples: 
+* `list`<br>
+  `setcomplete 2`<br>
+  Set the 2nd task as complete in the task manager.
+
+#### Listing all tasks due today : `list`
+Shows a list of all tasks due today in the task manager.<br>
+Format: `list`
+
+#### Listing all floating tasks : `listfloat`
+Shows a list of all floating tasks in the task manager.<br>
+Format: `listfloat`
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+#### Deleting a task : `delete`
+Deletes the specified task from the task manager.<br>
+Format: `delete TASK_INDEX`
 
-> Deletes the person at the specified `INDEX`. 
+> Deletes the task at the specified `TASK_INDEX`.<br>
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
 * `list`<br>
   `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
+  Deletes the 2nd task in the task manager.
+* `find report`<br> 
   `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  Deletes the 1st task in the results of the `find` command.
+  
+#### Finding all tasks containing any keyword in their name : `find`
+Finds tasks whose names contain any of the given keywords.<br>
+Format: `find KEYWORD [MORE_KEYWORDS]`
+
+> * The search is not case sensitive. e.g `complete` will match `Complete`
+> * The order of the keywords does not matter. e.g. `meeting John` will match `John meeting`
+> * Partial words will be matched e.g. `meet` will match `meeting`
+> * Tasks matching at least one keyword will be returned (i.e. `OR` search).
+    e.g. `meeting` will match `meeting with John`
+
+Examples: 
+* `find Highlight`<br>
+  Returns any tasks having names `Highlight` or `highlight`
+* `find product highlight showcase`<br>
+  Returns any task having names `product`, `highlight`, or `showcase`
+
+#### Finding all tasks within a date range : `finddate`
```

Comment:
same comment as above, can you think of providing one find command and use optional parameters for variations?